scribble-lib/scribble/acmart.rkt

```diff
@@ -9,7 +10,31 @@
          (for-syntax racket/base))
 
 (provide/contract
- [title (->* (pre-content?) (#:short pre-content?) content?)]
+ [title (->* (pre-content?)
+             (#:short pre-content?
+              #:tag (or/c string? (listof string?) #f)
+              #:tag-prefix (or/c string? module-path? #f)
+              #:style (or/c style? string? symbol? #f)
+              #:version (or/c string? #f)
+              #:date (or/c string? #f))
+             title-decl?)]
+ [author (->* () () #:rest (listof pre-content?)
```

Comment:
Good question. I air on the side of leaving both in because that's what all of the other `scribble/*` langs do. Even `scribble/lncs` and `scribble/sigplan`, both of which also have an authorinfo type of structure. But honestly, I'm not super picky, so whatever.